src/main/java/dev/esophose/playerparticles/particles/PParticle.java

```diff
@@ -33,6 +34,26 @@ public PParticle(Location location, double xOff, double yOff, double zOff, doubl
         this.directional = directional;
         this.overrideData = overrideData;
     }
+    /**
+     * The constructor with all the fancy parameters and override data for customization
+     *
+     * @param location The location to display the particle at
+     * @param xOff The offset for the x-axis
+     * @param yOff The offset for the y-axis
+     * @param zOff The offset for the z-axis
+     * @param speed The speed the particle will move at
+     * @param directional If the particle should use the x, y, and z offsets as directions instead
+     * @param size Size of the redstone particle
+     */
+    public PParticle(Location location, double xOff, double yOff, double zOff, double speed, boolean directional, float size) {
```

Comment:
I don't really like the approach that was taken here. This won't allow changing the color and size of dust and dust color transition particles at the same time. I don't think the size attribute really belongs in the PParticle class at all andshould be part of the overrideData object. Maybe come up with some other way of getting the size to be passed through the overrideData object instead.

Reply:
Using override data overrides the color chosen by the player.